src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  padding: $calendar-paddings;
+  max-width: $calendar-width;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-shrink: 0;
+    width: $day-width;
+    height: $day-height;
+
+    font-family: Arial,
+      Helvetica,
+      sans-serif;
+    font-size: 30px;
+
+    background-color: #ccc;
+    border: solid 1px #000;
+    cursor: pointer;
+
+    transition: background-color $transition-duration,
+      transform $transition-duration;
+
+    &:hover {
+      background-color: #FFBFCB;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through 31 {
+
+      &:nth-child(#{$i}) {
+
+        &::before {
+          content:'#{$i}';
+        }
+
+      }
+
+    }
+
+  }
+
+  @each $day, $offset in $days {
+
+    &--start-day-#{$day} {
+
+      :first-child {
+        margin-left: ($day-width + $days-gap-size ) * ($offset - $days-gap-size);
+      }
+
+    }
+
+  }
+
+  @for $i from 28 through 31 {
+
+    &--month-length-#{$i}
+
+    &__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+
+  }
+
 }
```

Comment:
Formatting is still not fixed 🤷
Why do you add so many blank lines? They are redundant